Functions.Templates/Templates/SqlOutputBinding-PowerShell/run.ps1

```diff
@@ -0,0 +1,28 @@
+<#
+Sample SQL Output Binding
+See https://aka.ms/sqlbindingsinput for more information about using this binding
+These tasks should be completed prior to running :
+    1. Update "commandText" in function.json - this should be the name of the table that you wish to upsert values to
+    2. Add an app setting named "SqlConnectionString" containing the connection string to use for the SQL connection
+    3. Change the bundle name in host.json to "Microsoft.Azure.Functions.ExtensionBundle.Preview" and the version to "[3.*, 4.0.0)"
```

Comment:
I'm sure we have this in other templates as well, but maybe we should be pointing users to 4.x bundles since 3.x won't be supported starting in December: https://learn.microsoft.com/en-us/azure/azure-functions/functions-versions?tabs=azure-cli%2Cwindows%2Cin-process%2Cv4&pivots=programming-language-csharp

Reply:
I can go ahead and update this and track a new issue to update (javascript, typescript). Python has noted that it should point to 4.x ie 
`Change the bundle name in host.json to "Microsoft.Azure.Functions.ExtensionBundle.Preview" and the version to "[4.*, 5.0.0)"
`